example-clients/capture_client.c

```diff
@@ -251,18 +254,16 @@ setup_ports (int sources, char *source_names[], jack_thread_info_t *info)
 			fprintf (stderr, "cannot connect input port %s to %s\n", jack_port_name (ports[i]), source_names[i]);
 			jack_client_close (info->client);
 			exit (1);
-		} 
+		}
 	}
 
 	info->can_process = 1;		/* process() can start, now */
 }
 
 int
 main (int argc, char *argv[])
-
 {
-	jack_client_t *client;
-	jack_thread_info_t thread_info;
+    jack_thread_info_t thread_info;
```

Comment:
tab vs spaces conflict here, will resolve after merging.

Reply:
I think a general code cleanup PR would be great once all the current changes are merged.
There is a lot of weird indentation, tab poisoning, inconsistent formatting, etc. going on :)